dashboard/lib/policies/registration.rb

```diff
@@ -0,0 +1,25 @@
+class Policies::Registration
+  NEW_USER_PERMITTED_PARAMS = [
```

Comment:
I'm not sure we need a separate constant for these parameters.
It should be sufficient to move them directly into `Services::PartialRegistration::UserBuilder#user_params`